db/seeds.rb

```diff
@@ -5,3 +5,17 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+require 'nokogiri'
+require 'open-uri'
+vocabulary = []
+data = Nokogiri::HTML(open("http://www.languagedaily.com/learn-german/vocabulary/common-german-words"))
+data.xpath("//tr/td").each do |cell|
+	if cell["class"] == "bigLetter"
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Use `next` to skip iteration.<br>Tab detected.